### PR TITLE
NMS-14634: Add alt text to Operation (pt. 3)

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `operation/deep-dive/bsm/business-service-map.adoc`, `/introduction.adoc`, `/map-functions.adoc`, `/operational-status.adoc`, `/reduce-functions.adoc`, `/root-cause-impact-analysis.adoc`
- Update `operation/deep-dive/topology/enlinkd/layer-2/bridge-discovery.adoc`, `/cdp-discovery.adoc`, `/lldp-discovery.adoc`
- Update `operation/deep-dive/topology/enlinkd/layer-3/is-is-discovery.adoc`, `/ospf-discovery.adoc`
- Update `operation/deep-dive/topology/graphml-asset-topology-provider.adoc`, `/topology.adoc`
- Update `operation/deep-dive/database-reports/database.adoc`
- Update `operation/deep-dive/database-reports/templates/avail-summary.adoc`, `/availability.adoc`, `/event.adoc`, `/grafana.adoc`, `/interface-avail.adoc`, `/maintenance-expired.adoc`, `/maintenance-strategy.adoc`, `/response-summary.adoc`, `/response-time.adoc`, `/serial.adoc`, `/snmp.adoc`, `/total-bytes.adoc`, `/traffic-rates.adoc`
- Update `operation/deep-dive/ticketing/introduction.adoc`
- Update `operation/deep-dive/workarounds/snmp.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://opennms.atlassian.net/browse/NMS-14634

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc [hunk 1]
```diff
@@ -1,158 +1,193 @@
...
 Conceptually, the topology is rendered as concentric sets.
 The Asset Topology Provider first searches all of the nodes with regions defined and creates a new level 0 graph node representing each region found.
-The Asset Topology Provider then searches within each region to find the building entries and creates a corresponding level 1 graph node for each building name found.
-Finally, the Asset Topology Provider creates layer 2 nodes corresponding to each {page-component-title} monitored node and places each in the correct building.
+It then searches within each region to find the building entries and creates a corresponding level 1 graph node for each building name found.
+Finally, it creates layer 2 nodes corresponding to each monitored node and places each in the correct building.
 
 {page-component-title}-monitored nodes that have empty region or building asset fields cannot be placed correctly in this topology.
-These nodes are shown in the diagram as unallocated nodes.
+These nodes are shown in the previous diagram as unallocated nodes.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc:36]:
```suggestion
The previous diagram shows these nodes as unallocated nodes.
```

File: docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc [hunk 2]
```diff
@@ -174,139 +209,151 @@ The configuration file contains each of the graph definitions as properties in t
...
-| (Uses the default settings if a particular setting is not included in the command.) +
--l, --label : Asset topology label (shows in topology menu) (Default: asset) +
--i, --providerId : Unique providerId of asset topology (Default: Asset Topology Provider) +
--f, --filter : Optional node filter (Default: empty filter; in other words, allow all nodes) +
--a, --assetLayers : Comma-separated list of asset layers (Default: asset-region,asset-building,asset-rack) +
--p, --preferredLayout : Preferred Layout (Default: Grid Layout) +
--b, --breadcrumbStrategy : Breadcrumb strategy (Default: SHORTEST_PATH_TO_ROOT) +
-
-If you simply type `asset-topology:create`, a default topology with providerId asset is created.
+Uses the default settings if a tag isn't included in the command.^1^
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc:256]:
```suggestion
Uses the default settings if a tag is not included in the command.^1^
```

File: docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc [hunk 3]
```diff
@@ -174,139 +209,151 @@ The configuration file contains each of the graph definitions as properties in t
...
-(If one graph fails, the command tries to complete the rest of the definitions.)
+If one graph fails, the command tries to complete the rest of the definitions.
 |
 |===
 
-== Create asset-based topologies using {page-component-title} events
-
-The Asset Topology Provider listens for events that trigger the generation and installation or removal of topologies.
-The Asset Topology Provider events are defined in the file:
+. If you simply type `asset-topology:create`, a default topology is created using `providerId`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/topology/graphml-asset-topology-provider.adoc:290]:
```suggestion
. If you type `asset-topology:create`, a default topology is created using `providerId`.
```

File: docs/modules/operation/pages/deep-dive/topology/topology.adoc [hunk 4]
```diff
@@ -6,93 +6,84 @@ This section describes how to configure the topology map.
...
+| Defines the auto-refresh interval, in seconds.
 | Integer
 | 60
 
 | hiddenCategoryPrefix
-| A string that enables hiding categories. For example a value of `server` will hide all categories starting with `server`.
-Be aware, that this setting is case sensitive, so `Servers` will not be shown.
-The resolution is enabled only if no longitude/latitude information is available.
+| Hides categories based on a defined prefix.
+For example, a value of `server` will hide all categories starting with "server".
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/topology/topology.adoc:33]:
```suggestion
For example, a value of `server` hides all categories starting with "server".
```

File: docs/modules/operation/pages/deep-dive/topology/topology.adoc [hunk 5]
```diff
@@ -6,93 +6,84 @@ This section describes how to configure the topology map.
...
-Enable them by changing the `logmsg` property on the `uei.opennms.org/internal/topology/linkDown` and `uei.opennms.org/internal/topology/linkUp` events from `donotpersist` to `logndisplay`.
-These events are automatically generated by the default set of event translator rules that process incoming SNMP Link Up & Down traps.
+Enable them by setting the `logmsg` property in the `uei.opennms.org/internal/topology/linkDown` and `uei.opennms.org/internal/topology/linkUp` events to `logndisplay`.
+These events are automatically generated by the default set of event translator rules that process incoming SNMP link up and down traps.
 
 == Icons
 
-Each Vertex on the topology map is represented by an icon.
-The default icon is configured in the icon mapping file: `$\{OPENNMS_HOME}/etc/org.opennms.features.topology.app.icons.<topology-namespace>.cfg`.
+Each vertex on the topology map is represented by an icon.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/topology/topology.adoc:58]:
```suggestion
An icon represents each vertex on the topology map.
```

File: docs/modules/operation/pages/deep-dive/workarounds/snmp.adoc [hunk 6]
```diff
@@ -1,43 +1,101 @@
 
 = Overriding SNMP Client Behavior
 
-By default, the SNMP subsystem in {page-component-title} does not treat any link:https://tools.ietf.org/html/rfc3416[RFC 3416] `error-status` as fatal.  
-Instead, it will attempt to continue the request, if possible.  
-However, only a subset of errors will cause {page-component-title}'s SNMP client to attempt retries.
+By default, the SNMP subsystem in {page-component-title} does not treat any https://tools.ietf.org/html/rfc3416[RFC 3416] `error-status` as fatal.
+Instead, it will attempt to continue the request, if possible.
+Only a subset of errors will cause {page-component-title}'s SNMP client to attempt retries, however.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/workarounds/snmp.adoc:6]:
```suggestion
Only a subset of errors will cause {page-component-title}'s SNMP client to attempt retries.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Nice work. A few nits. Thanks!

File: docs/modules/operation/pages/deep-dive/bsm/map-functions.adoc [hunk 7]
```diff
@@ -1,13 +1,13 @@
 
 = Map Functions
 
-The map functions define how the severity of the edge will be used in the reduce function of the parent when calculating the operational status.
+The map function defines how the severity of an edge is used in the parent system's reduce function when the operational status is calculated.
```
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/deep-dive/bsm/map-functions.adoc:4]: I would assume in this case that it's Horizon/Meridian, but I'll ask around. :)

[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/deep-dive/bsm/map-functions.adoc:4]: I think so too. We can replace XX with {page-component-title}.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Everything looks good, just the outstanding comment about the map function.

File: docs/modules/operation/pages/deep-dive/bsm/map-functions.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/deep-dive/bsm/map-functions.adoc:4]:
```suggestion
The map function defines how to use the severity of an edge in the parent system's reduce function when {page-component-title} calculates the operational status of the business service.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**